lab_1_keywords_tfidf/main.py

```diff
@@ -66,7 +99,26 @@ def get_top_n(frequencies: dict[str, Union[int, float]], top: int) -> Optional[l
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if (isinstance(frequencies, dict) and frequencies != {} and isinstance(top, int) and top > 0
```

Comment:
revert the condition and return None immediately

Reply:
done